packages/notebook/src/browser/contributions/notebook-cell-actions-contribution.ts

```diff
@@ -190,10 +190,10 @@ export class NotebookCellActionContribution implements MenuContribution, Command
         });
 
         // Notebook Cell extra execution options
-        menus.registerIndependentSubmenu(NotebookCellActionContribution.CONTRIBUTED_CELL_EXECUTION_MENU,
-            nls.localizeByDefault('More...'),
-            { role: CompoundMenuNodeRole.Flat, icon: codicon('chevron-down') });
-        menus.getMenu(NotebookCellActionContribution.CODE_CELL_SIDEBAR_MENU).addNode(menus.getMenuNode(NotebookCellActionContribution.CONTRIBUTED_CELL_EXECUTION_MENU));
+        // menus.registerIndependentSubmenu(NotebookCellActionContribution.CONTRIBUTED_CELL_EXECUTION_MENU,
+        //     nls.localizeByDefault('More...'),
+        //     { role: CompoundMenuNodeRole.Flat, icon: codicon('chevron-down') });
+        // menus.getMenu(NotebookCellActionContribution.CODE_CELL_SIDEBAR_MENU).addNode(menus.getMenuNode(NotebookCellActionContribution.CONTRIBUTED_CELL_EXECUTION_MENU));
```

Comment:
If it is not needed anymore it could be removed.

Reply:
its not like its not needed. We will have to add it again at a later point in time, but since its currently not working i removed it for now